demo/atom/button/themes/carfactory.scss

```diff
@@ -0,0 +1,2 @@
+@import '../../../../themes/carfactory';
```

Comment:
Themes are defined in /theme root folder so they can me maintained easily in the future